packages/react-vapor/src/components/input/examples/InputExamples.1.use.md

```diff
@@ -0,0 +1,20 @@
+## Guidelines
+
+### Restrictions & Size
+
+-   Users should be informed of any input restrictions such as character types or string lenght.
+-   The text input width should give a hint on the expected size of the content to enter. Try to forsee the usual string length so as to avoid hiding parts of the entered strings. The text input field should be large enough to see the whole value in most cases.
+-   Avoid setting a character limit or make sure to set it high enough to avoid constraining most user entry. If this is not possible, let users know how many characters are left.
```

Comment:
I'm curious where we should put this restriction and whether we should only show it after a certain threshold (like 10 or less characters left)

Reply:
Showing it after a certain threshold does not seem like a bad idea to me at all :)